packages/api-request-builder/test/query-id.spec.js

```diff
@@ -34,12 +34,29 @@ describe('queryId', () => {
     expect(service.params.customerId).toBe('myCustomer')
   })
 
+  test('should set the orderNumber param', () => {
+    service.byOrderNumber(123)
+    expect(service.params.orderNumber).toBe(123)
+  })
+
   test('should throw if customerId is missing', () => {
     expect(() => service.byCustomerId()).toThrow(
       /Required argument for `byCustomerId` is missing/
     )
   })
 
+  test('should throw if orderNumber is missing', () => {
+    expect(() => service.byOrderNumber()).toThrow(
+      /Required argument for `byOrderNumber` is missing or invalid/
```

Comment:
```suggestion
      /Required argument for `byOrderNumber` is missing/
```

another ocd nitpick from me 😆

Reply:
lol, this is fixed now